### ISSUE TITLE
Import Error

### ISSUE THREAD

**Issue by Pranavmungath** [JarryShaw/PyPCAPKit#116]
Import error in python 3.8 for latest version of pcapkit

Traceback (most recent call last):

  File "C:\Users\MPN8KOR\Documents\venv\_venv3.8\Lib\site-packages\pcapkit\untitled2.py", line 1, in <module>
    import pcapkit

  File "C:\Users\MPN8KOR\Documents\venv\_venv3.8\lib\site-packages\pcapkit\__init__.py", line 83, in <module>
    from pcapkit.foundation.registry import *

  File "C:\Users\MPN8KOR\Documents\venv\_venv3.8\lib\site-packages\pcapkit\foundation\__init__.py", line 13, in <module>
    from pcapkit.foundation.extraction import Extractor

  File "C:\Users\MPN8KOR\Documents\venv\_venv3.8\lib\site-packages\pcapkit\foundation\extraction.py", line 22, in <module>
    from pcapkit.corekit.infoclass import Info

  File "C:\Users\MPN8KOR\Documents\venv\_venv3.8\lib\site-packages\pcapkit\corekit\__init__.py", line 15, in <module>
    from pcapkit.corekit.infoclass import Info

  File "C:\Users\MPN8KOR\Documents\venv\_venv3.8\lib\site-packages\pcapkit\corekit\infoclass.py", line 25, in <module>
    class Info(collections.abc.Mapping[str, VT], Generic[VT]):

TypeError: 'ABCMeta' object is not subscriptable

**Comment by claw16** [comment]
same here, any solutions? Thanks!

```---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-6-406dd2651dc1> in <module>
----> 1 from pcapkit import extract
      2 file_path = r"D:\Users\CL\Downloads\cl_work\pcap_parser\pcap_logs\FCAWL_20220920_RGVIN647_FTP_RG55C3_GR_TRY_sb_091348_002.pcap"
      3 out_path = r"D:\Users\CL\Downloads\cl_work\pcap_parser\pcap_logs\FCAWL_20220920_RGVIN647_FTP_RG55C3_GR_TRY_sb_091348_002.json"
      4 json = extract(fin=file_path, fout=out_path, format='json', extension=False)

~\Miniconda3\envs\data_env\lib\site-packages\pcapkit\__init__.py in <module>
     81                          exception=BaseError, strict=False)
     82 
---> 83 from pcapkit.foundation.registry import *
     84 from pcapkit.interface import *
     85 from pcapkit.protocols import *

~\Miniconda3\envs\data_env\lib\site-packages\pcapkit\foundation\__init__.py in <module>
     11 
     12 """
---> 13 from pcapkit.foundation.extraction import Extractor
     14 from pcapkit.foundation.traceflow import TraceFlow
     15 from pcapkit.foundation.reassembly import *

~\Miniconda3\envs\data_env\lib\site-packages\pcapkit\foundation\extraction.py in <module>
     20 
     21 from pcapkit.const.reg.linktype import LinkType as RegType_LinkType
---> 22 from pcapkit.corekit.infoclass import Info
     23 from pcapkit.protocols.misc.pcap.frame import Frame
     24 from pcapkit.protocols.misc.pcap.header import Header

~\Miniconda3\envs\data_env\lib\site-packages\pcapkit\corekit\__init__.py in <module>
     13 
     14 """
---> 15 from pcapkit.corekit.infoclass import Info
     16 from pcapkit.corekit.multidict import MultiDict, OrderedMultiDict
     17 from pcapkit.corekit.protochain import ProtoChain

~\Miniconda3\envs\data_env\lib\site-packages\pcapkit\corekit\infoclass.py in <module>
     23 
     24 
---> 25 class Info(collections.abc.Mapping[str, VT], Generic[VT]):
     26     """Turn dictionaries into :obj:`object` like instances.
     27 

TypeError: 'ABCMeta' object is not subscriptable
```

**Comment by BrianMiller793** [comment]
Bug is only present in 0.16.0 and above.  Bug is not in 0.15.5 and below.

**Comment by BrianMiller793** [comment]
The problem is originating from `pcapkit/corekit/infoclass.py`, in the `class Info(collections.abc.Mapping[str, VT], Generic[VT]):` declaration.  I'm not familiar with multiple inheritance like this, or Mapping or Generic, so I'm flailing beyond this.  The Google crowd wisdom recommends using `typing.Mapping` instead of `collections.abc.Mapping`.

**Comment by JarryShaw** [comment]
Should've been fixed in version 0.16.3
